utils/hdparm.go

```diff
@@ -44,7 +44,7 @@ func (h *Hdparm) ListFeatures(device string) ([]byte, error) {
 }
 
 // nolint:gocyclo // line parsing is cyclomatic
-func (h *Hdparm) parseHdparmFeatures(device string) ([]hdparmDeviceFeatures, error) {
+func (h *Hdparm) ParseHdparmFeatures(device string) ([]hdparmDeviceFeatures, error) {
```

Comment:
Does this method need to be exported, should it be lower cased instead?

Reply:
The plan was to export it in both nvme and hdparm if we were calling from lsblk and using that for the collection, but we decided against it. I've reverted it.